### PR TITLE
CLDR-15570 BRS Update to emoji 15

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15570

Following https://cldr.unicode.org/development/generate-emoji-paths

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/annotations/en.xml [hunk 1]
```diff
@@ -3842,5 +3842,49 @@ annotations.
 		<annotation cp="³" type="tts">superscript three</annotation>
 		<annotation cp="µ">measure | micro sign</annotation>
 		<annotation cp="µ" type="tts">micro sign</annotation>
+<!-- Generated lines from Emoji Data v15, using GenerateCldrData.java-->
+		<annotation cp="🫨">earthquake | face | shaking | shock | vibrate</annotation> 	<!-- 1FAE8 -->
+		<annotation cp="🫨" type="tts">shaking face</annotation>
+		<annotation cp="🩵">cyan | heart | light blue | teal</annotation> 	<!-- 1FA75 -->
+		<annotation cp="🩵" type="tts">light blue heart</annotation>
```
[the remark this answers is not in the document]
Comment (macchiati) [reply common/annotations/en.xml:3849]: That is in the description:

> Following https://cldr.unicode.org/development/generate-emoji-paths

(It is generated from the unicodetools repo)

[the remark this answers is not in the document]
Comment (btangmu) [reply common/annotations/en.xml:3849]: found it: https://github.com/unicode-org/unicodetools/blob/main/unicodetools/src/main/java/org/unicode/tools/emoji/GenerateCldrData.java

**Review by btangmu — APPROVED** [review]

File: common/annotations/en.xml [hunk 2]
```diff
@@ -3842,5 +3842,49 @@ annotations.
...
+		<annotation cp="🫎">animal | antlers | elk | mammal</annotation> 	<!-- 1FACE -->
+		<annotation cp="🫎" type="tts">moose</annotation>
+		<annotation cp="🫏">animal | ass | burro | mammal | mule | stubborn</annotation> 	<!-- 1FACF -->
+		<annotation cp="🫏" type="tts">donkey</annotation>
+		<annotation cp="🪽">angelic | aviation | bird | flying | mythology</annotation> 	<!-- 1FABD -->
+		<annotation cp="🪽" type="tts">wing</annotation>
+		<annotation cp="🪿">bird | fowl | honk | silly</annotation> 	<!-- 1FABF -->
+		<annotation cp="🪿" type="tts">goose</annotation>
+		<annotation cp="🐦‍⬛">bird | black | crow | raven | rook</annotation> 	<!-- 1F426 200D 2B1B -->
+		<annotation cp="🐦‍⬛" type="tts">black bird</annotation>
```
Comment (nedley) [inline common/annotations/en.xml:3867]: Originally BLACK BIRD came before GOOSE, is there a reason why this changed?

**Comment by macchiati** [comment]
I don't know why; we can correct it if you want.

**Comment by nedley** [comment]
> I don't know why; we can correct it if you want.

I’m making the correction as part of https://github.com/unicode-org/emoji/pull/24 in order to be consistent with existing charts, feel free to re-import the change here.

**Pull request merged**